testcase/var_visibility_prefix_invalid_file.mk

```diff
@@ -0,0 +1,19 @@
+FOO := foo
+BAR := bar
+$(KATI_visibility_prefix FOO, Makefile)
+$(KATI_visibility_prefix BAR, )
+$(KATI_visibility_prefix BAZ, baz)
+
+VAR0 := $(FOO)
+VAR1 := $(BAR)
+VAR2 := $$(BAZ)
+VAR3 = $$(BAZ)
+VAR4 = $($(BAZ))
+VAR5 := $($(BAZ))
```

Comment:
VAR3, VAR4, and VAR5 are not being tested because the makefile errored out at VAR2. You should move them into separate tests.

Reply:
From our discussion, VAR2/VAR3 aren't referenced due to the double dollar sign, and VAR4 isn't because the recursive expansion means it's not expanded.

That VAR4 case is a bug in my eyes, we should also check for usages via recursive expansion. This may require expanding recursive assignments immediately but then throwing away the results.